app/controllers/likes_controller.rb

```diff
@@ -1,5 +1,6 @@
 class LikesController < ApplicationController
   before_action :set_like, only: %i[ show edit update destroy ]
+  before_action { authorize( @likes || Like) }
```

Comment:
I would stay consistent and use the singular `@like` like you did elsewhere. Not an issue here yet since the like function is not fully implemented